Honeybadger/screenboard.json

```diff
@@ -0,0 +1 @@
+{}
```

Comment:
@joshuap Thanks for sending this along.  To confirm are you sending any metrics to Datadog at the moment that might make sense to show on a screenboard? Or is it just events at the moment?